### ISSUE TITLE
[MU4 Task]More control over colours

### ISSUE THREAD

**Issue by LaxVox** [musescore/MuseScore#11525]
Add a way to easily change the colours of the score (with an ability to invert the score to "dark mode" manually and not to eye-straining white against full black). Allow users to customize colour on more UI elemenst

**Comment by bkunda** [comment]
I'm not sure I fully understand the request here. If you have a specific part of the UI that is concerning, then please identify it (preferably with screenshots). 

MuseScore 4 does have very carefully designed dark, light, and high contrast modes, and offers the facility to change the background and paper colours to virtually anything the user wants (See Preferences > Appearance). All of the colours in high contrast mode are also completely customisable, so it is technically possible to customise the UI any way you want. 

Closing this for now because we won't be offering further UI customisation functionality at this stage.

**Comment by bkunda** [comment]
It's an intriguing idea to be able to customise the note and stave colour. I don't want to suggest ruling this idea out completely, but it is such a fine level of customisability that we can't factor it in to the MS4 release I'm afraid (we simply have too many other far more pressing issues to implement and test).

**Comment by Tantacrul** [comment]
Also, the score **can** now be inverted in preferences

**Comment by LaxVox** [comment]
> Also, the score **can** now be inverted in preferences

I just downloaded the latest nightly build and from what I gathered, the only way a score can be is black&white, and while it is ok in light mode, when inverted, the contrast of white elements on the blackest background is just too strong and I don't think it is usable at all. There should be a way to at least change the paper colour to a lighter shade of grey when inverted. Moreover, it's already implemented by default but (for reasons unknown to me) disabled in inverted mode.
